nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -511,7 +533,6 @@ in {
         pools.nextcloud = {
           user = "nextcloud";
           group = "nextcloud";
-          phpOptions = phpOptionsStr;
```

Comment:
Have you confirmed whether this is actually fine?

Reply:
I took this commit from @aanderse . I don't know whether it is correct. It worked on my server, but I didn't tweak this setting.

Reply:
@Ma27 as you can see it is applied twice. I looked and confirmed duplicate config was generated.